views/dashboard-application.hbs

```diff
@@ -47,49 +47,51 @@
 
         {{else}}
 
-            <div class='top-box'></div>
-
-            <div class='ideahacks-form application-form container-fluid'>
-                <div class='row'>
-                    <div class='mainHeader col-md-12 text-center'>
-                        <h1>IDEAHACKS: HOME APPLICATIONS</h1>
-                    </div>
+            <div id="intro">
+                <div class="intro_view">
+                    <img id="application_logo" src="/img/application/applicationlogo.png" alt="application logo">
+                    <img id="intro_graphics" src="/img/application/application.png" alt="application graphics">
```

Comment:
On small screens, the logo runs totally over the graphic and is impossible to read. Can you do the same thing you did for the history page so it doesn't do that? Or you can try what Tim did on the team page, which seems to work better

Reply:
Hmm... I actually think I used the same code as my history page but I'm not sure why it's working better in history page and not here.